rgw/COSBench/haproxy.cfg

```diff
@@ -93,7 +94,8 @@ listen  stats   0.0.0.0:1936
 #---------------------------------------------------------------------
 # main frontend which proxys to the backends
 #---------------------------------------------------------------------
-frontend  main *:5000
+frontend  main
+    bind    *:5000
```

Comment:
same here, looks like a format issue

Reply:
I have verified these parameters on my environment and these parameters are working in my environment